app/web_conn.go

```diff
@@ -28,11 +29,11 @@ const (
 )
 
 type WebConn struct {
+	sessionExpiresAt          int64
```

Comment:
I'm also going to recommend a comment here mentioning that this should be kept first.